src/GrumPHP/Exception/FileNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace GrumPHP\Exception;
+
+/**
+ * @author Igor Mukhin <igor.mukhin@gmail.com>
+ */
+class FileNotFoundException extends RuntimeException
+{
+    /**
+     * @param string $path
+     */
+    public function __construct($path)
+    {
+        parent::__construct(sprintf(
+            "File '%s' doesn't exists.",
```

Comment:
Can you invert the single/double quote usage here? We tend to use single quotes for strings in other parts of the codebase. Also, any chance this fits on one line?